### PR TITLE
update introduction prose

### PR DESCRIPTION
**Pull request by js-kyle**
The repo introduction needs a little bit of improvement for readability, so I have proposed some changes. These won't be perfect either but lets use it as a point of discussion?

Mainly keen to improve this as it is often the first thing someone will read when visiting

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,15 +22,15 @@ Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chines
 
 <br/>
 
-# Welcome! 3 Things You Ought To Know First:
+# Welcome! First, three things you should know:
 
-**1. When you read here, you in fact read dozens of the best Node.js articles -** this is a summary and curation of the top-ranked content on Node.js best practices
+**1. The content presented here is a curation of the best Node.js articles -** this repository is a summary and curation of the top-ranked content on Node.js best practices, as well as content written here by collaborators
```
Comment (sagirk) [inline README.md]: nit: The idea of 'curation' is repeated in the bullet and its description. The original text, in spirit, seemed more 'DRY'.

```suggestion
**1. You are, in fact, reading dozens of the best Node.js articles -** this repository is a summary and curation of the top-ranked content on Node.js best practices, as well as content written by collaborators
```

File: README.md [hunk 2]
```diff
@@ -22,15 +22,15 @@ Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chines
...
+**2. It is the largest compilation, and it is growing every week -** currently, more than 70 best practices, style guides, and architectural tips are presented. New issues and pull requests are created every day to keep this live book updated. We'd love to see you contributing here, whether that is fixing code mistakes, helping with translations, or suggesting brilliant new ideas. See our [writing guidelines here](/.operations/writing-guidelines.md)
 
-**3. Most bullets have additional info -** nearby most best practice bullets you'll find **🔗Read More** link that will present you with code examples, quotes from selected blogs and more info
+**3. Most best practices have additional info -** most best practices include a **🔗Read More** link that expands on the practice with code examples, quotes from selected blogs and more information
```
Comment (sagirk) [inline README.md]: nit: 'practices' occurs thrice; replacing one occurrence with 'bullets' seems to flow better.

```suggestion
**3. Most best practices have additional info -** most bullets include a **🔗Read More** link that expands on the practice with code examples, quotes from selected blogs and more information
```

File: README.md [hunk 3]
```diff
@@ -22,15 +22,15 @@ Read in a different language: [![CN](/assets/flags/CN.png)**CN**](/README.chines
 
 <br/>
 
-# Welcome! 3 Things You Ought To Know First:
+# Welcome! First, three things you should know:
```
[the remark this answers is not in the document]
Comment (sagirk) [reply README.md]: Yup, we can keep it as is — "3 Things You Ought To Know First".

**Comment by i0natan** [comment]
@js-kyle Not sure where this stand, your call on all edits, just ensuring you're not awaiting for any input from us

**Comment by i0natan** [comment]
🎆

**Pull request merged**